src/engine/renderer/tr_init.cpp

```diff
@@ -77,7 +77,9 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 		Util::ordinal(realtimeLightingRenderer_t::TILED),
 		Util::ordinal(realtimeLightingRenderer_t::LEGACY),
 		Util::ordinal(realtimeLightingRenderer_t::TILED) );
-	Cvar::Cvar<bool> r_realtimeLighting( "r_realtimeLighting", "enable realtime light rendering", Cvar::NONE, true );
+	Cvar::Cvar<bool> r_realtimeLighting( "r_realtimeLighting", "Enable realtime light rendering", Cvar::NONE, true );
+	Cvar::Range<Cvar::Cvar<int>> r_realtimeLightLayers( "r_realtimeLightLayers", "Dynamic lights per tile, each layer holds 16 lights",
```

Comment:
This seems to be missing a word. Like "Realtime light *layers* per tile"

Reply:
Oh, right. Fixed now.